### PR TITLE
add apollo team introduction and community releated contents

### PR DESCRIPTION
**Pull request by nobodyiam**
## What's the purpose of this PR

add apollo team introduction and community releated contents

Reference issues: #3684 #3705 #3706 #3707 #3708 #3709

## Brief changelog

1. add team introduction page
2. add links to community governance

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Read the [Contributing Guide](https://github.com/ctripcorp/apollo/blob/master/CONTRIBUTING.md) before making this pull request.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [x] Write necessary unit tests to verify the code.
- [x] Run `mvn clean test` to make sure this pull request doesn't break anything.

### CODE REVIEW

File: docs/en/community/team.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+| Github ID  | Name       | Organization |
+| ---------- | ---------- | ------------ |
+| JaredTan95 | Jared Tan  | DaoCloud     |
+| kezhenxu94 | Zhenxu Ke  | Tetrate      |
+| nobodyiam  | Jason Song | Ant Group    |
+
+### Committer
+| Github ID   | Name          | Organization |
+| ----------- | ------------- | ------------ |
+| Anilpole    | Xiaoquan Wang |              |
```
Comment (nobodyiam) [inline docs/en/community/team.md:22]: @Anilople @klboke @nisiyong @vdisk-group 

please help to update the corresponding content.

File: pom.xml [hunk 2]
```diff
@@ -99,6 +99,62 @@
 				<role>Developer</role>
 			</roles>
 		</developer>
+		<developer>
+			<id>JaredTan95</id>
+			<url>https://github.com/JaredTan95</url>
+			<name>Jared Tan</name>
+			<email>TBD</email>
```
Comment (nobodyiam) [inline pom.xml:106]: @JaredTan95 please help to update the corresponding content.

File: pom.xml [hunk 3]
```diff
@@ -99,6 +99,62 @@
...
+			<roles>
+				<role>Project Management Committee Member</role>
+				<role>Developer</role>
+			</roles>
+		</developer>
+		<developer>
+			<id>Anilpole</id>
+			<url>https://github.com/Anilpole</url>
+			<name>Xiaoquan Wang</name>
+			<email>TBD</email>
```
Comment (nobodyiam) [inline pom.xml:118]: @Anilople please help to update the corresponding content.

File: pom.xml [hunk 4]
```diff
@@ -99,6 +99,62 @@
...
+			<organizationUrl>TBD</organizationUrl>
+			<roles>
+				<role>Developer</role>
+			</roles>
+		</developer>
+		<developer>
+			<id>klboke</id>
+			<url>https://github.com/klboke</url>
+			<name>Kailing Chen</name>
+			<email>TBD</email>
```
Comment (nobodyiam) [inline pom.xml:129]: @klboke please help to update the corresponding content.

File: pom.xml [hunk 5]
```diff
@@ -99,6 +99,62 @@
...
+			<organizationUrl>TBD</organizationUrl>
+			<roles>
+				<role>Developer</role>
+			</roles>
+		</developer>
+		<developer>
+			<id>nisiyong</id>
+			<url>https://github.com/nisiyong</url>
+			<name>Stephen Ni</name>
+			<email>TBD</email>
```
Comment (nobodyiam) [inline pom.xml:140]: @nisiyong please help to update the corresponding content.

File: pom.xml [hunk 6]
```diff
@@ -99,6 +99,62 @@
...
+			<organizationUrl>TBD</organizationUrl>
+			<roles>
+				<role>Developer</role>
+			</roles>
+		</developer>
+		<developer>
+			<id>vdisk-group</id>
+			<url>https://github.com/vdisk-group</url>
+			<name>Lvqiu Ye</name>
+			<email>TBD</email>
```
Comment (nobodyiam) [inline pom.xml:151]: @vdisk-group please help to update the corresponding content.

File: docs/en/community/team.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+| klboke      | Kailing Chen  |              |
```
Comment (klboke) [inline docs/en/community/team.md:23]:
```suggestion
| klboke      | Kailing Chen  |  TapTap          |
```

File: pom.xml [hunk 8]
```diff
@@ -99,6 +99,62 @@
...
+			<roles>
+				<role>Developer</role>
+			</roles>
+		</developer>
+		<developer>
+			<id>klboke</id>
+			<url>https://github.com/klboke</url>
+			<name>Kailing Chen</name>
+			<email>TBD</email>
+			<organization>TBD</organization>
```
Comment (klboke) [inline pom.xml]:
```suggestion
			<organization>TapTap</organization>
```

File: pom.xml [hunk 9]
```diff
@@ -99,6 +99,62 @@
...
+			<organizationUrl>TBD</organizationUrl>
```
Comment (klboke) [inline pom.xml]:
```suggestion
			<organizationUrl>https://github.com/TapTap</organizationUrl>
```

File: pom.xml (hunk 4 above)
Comment (klboke) [inline pom.xml]:
```suggestion
			<email>632104866@qq.com</email>
```

**Review by klboke — COMMENTED** [review]
updated

File: docs/en/community/team.md (hunk 1 above)
Comment (Anilople) [inline docs/en/community/team.md:22]:
```suggestion
| Anilpole    | Xiaoquan Wang | Bank         |
```

File: docs/zh/community/team.md [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+| Github ID  | Name       | Organization |
+| ---------- | ---------- | ------------ |
+| JaredTan95 | Jared Tan  | DaoCloud     |
+| kezhenxu94 | Zhenxu Ke  | Tetrate      |
+| nobodyiam  | Jason Song | Ant Group    |
+
+### Committer
+| Github ID   | Name          | Organization |
+| ----------- | ------------- | ------------ |
+| Anilpole    | Xiaoquan Wang |              |
```
Comment (Anilople) [inline docs/zh/community/team.md:22]:
```suggestion
| Anilople    | Xiaoquan Wang | Bank         |
```

**Comment by Anilople** [comment]
How about add **Community** or **team** or **About Us** in navbar?i.e https://github.com/ctripcorp/apollo/blob/master/docs/_navbar.md

Example:
* Team in https://skywalking.apache.org/
* Community in https://shardingsphere.apache.org/

**Comment by Anilople** [comment]
I have read the CLA Document and I hereby sign the CLA

File: docs/zh/community/team.md [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+| Github ID  | Name       | Organization |
+| ---------- | ---------- | ------------ |
+| JaredTan95 | Jared Tan  | DaoCloud     |
+| kezhenxu94 | Zhenxu Ke  | Tetrate      |
+| nobodyiam  | Jason Song | Ant Group    |
+
+### Committer
+| Github ID   | Name          | Organization |
+| ----------- | ------------- | ------------ |
+| Anilople    | Xiaoquan Wang | Bank         |
```
Comment (Anilople) [inline docs/zh/community/team.md:22]:
```suggestion
| Anilople    | Xiaoquan Wang | Some Bank    |
```

**Review by Anilople — APPROVED** [review]

**Comment by nisiyong** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by vdisk-group — DISMISSED** [review]
updated

**Comment by nobodyiam** [comment]
> How about add **Community** or **team** or **About Us** in navbar?i.e https://github.com/ctripcorp/apollo/blob/master/docs/_navbar.md
> 
> Example:
> 
> * Team in https://skywalking.apache.org/
> * Community in https://shardingsphere.apache.org/

This is a much better idea!

**Comment by JaredTan95** [comment]
All contributors have signed the CLA and welcome aboard~

**Comment by JaredTan95** [comment]
> > How about add **Community** or **team** or **About Us** in navbar?i.e https://github.com/ctripcorp/apollo/blob/master/docs/_navbar.md
> > Example:
> > 
> > * Team in https://skywalking.apache.org/
> > * Community in https://shardingsphere.apache.org/
> 
> @Anilople
> 
> This is a great idea! I tried to [move the community menu to _navbar.md](https://github.com/ctripcorp/apollo/pull/3713/commits/4dcf5a8cc0db8c720d0f8847618bd5b4ea11034e), it looks much better!
> However, I don't know how to make the menu adapt for the translations automatically, e.g. when user change the translation to Chinese, then we display the Chinese menu. Would you please help?

This could be done in separate PR, Let's merge this PR?

**Review by JaredTan95 — APPROVED** [review]
LGTM

**Comment by nobodyiam** [comment]
> > > How about add **Community** or **team** or **About Us** in navbar?i.e https://github.com/ctripcorp/apollo/blob/master/docs/_navbar.md
> > > Example:
> > > 
> > > * Team in https://skywalking.apache.org/
> > > * Community in https://shardingsphere.apache.org/
> > 
> > 
> > @Anilople
> > This is a great idea! I tried to [move the community menu to _navbar.md](https://github.com/ctripcorp/apollo/pull/3713/commits/4dcf5a8cc0db8c720d0f8847618bd5b4ea11034e), it looks much better!
> > However, I don't know how to make the menu adapt for the translations automatically, e.g. when user change the translation to Chinese, then we display the Chinese menu. Would you please help?
> 
> This could be done in separate PR, Let's merge this PR?

@JaredTan95 I have reverted the latest change on the navbar.md, please review again.

File: docs/zh/community/team.md [hunk 12]
```diff
@@ -0,0 +1,40 @@
...
+
+Apollo 团队由 Member 和 Contributor 组成。Member 可以直接访问 Apollo 项目的源代码并基于代码库积极演进。Contributor 通过向 Member 提交补丁和建议来改善项目，项目的贡献者数量是没有限制的。无论是进行小规模的清理，提交新的功能或其它形式的贡献，都将受到极大的赞赏。
+
+有关社区治理模型的更多信息，请参考[GOVERNANCE.md](https://github.com/ctripcorp/apollo/blob/master/GOVERNANCE.md)。
+
+## Member
+
+Member 包括 PMC 成员和 Committer，该列表按字母顺序排列。
+
+### Project Management Committee(PMC)
```
Comment (JaredTan95) [inline docs/zh/community/team.md:11]: What about translating it as "### 项目管理委员会(PMC)"?

File: docs/zh/community/team.md [hunk 13]
```diff
@@ -0,0 +1,40 @@
+# Apollo 团队
+
+Apollo 团队由 Member 和 Contributor 组成。Member 可以直接访问 Apollo 项目的源代码并基于代码库积极演进。Contributor 通过向 Member 提交补丁和建议来改善项目，项目的贡献者数量是没有限制的。无论是进行小规模的清理，提交新的功能或其它形式的贡献，都将受到极大的赞赏。
+
+有关社区治理模型的更多信息，请参考[GOVERNANCE.md](https://github.com/ctripcorp/apollo/blob/master/GOVERNANCE.md)。
+
+## Member
```
Comment (JaredTan95) [inline docs/zh/community/team.md:7]: What about translating it as "## 成员(Member)"?

File: docs/zh/community/team.md [hunk 14]
```diff
@@ -0,0 +1,40 @@
...
+
+### Project Management Committee(PMC)
+
+| Github ID  | Name       | Organization |
+| ---------- | ---------- | ------------ |
+| JaredTan95 | Jared Tan  | DaoCloud     |
+| kezhenxu94 | Zhenxu Ke  | Tetrate      |
+| nobodyiam  | Jason Song | Ant Group    |
+
+### Committer
```
Comment (JaredTan95) [inline docs/zh/community/team.md:19]: the same as above ，`### 提交者(Committer)`?

File: docs/zh/community/team.md [hunk 15]
```diff
@@ -0,0 +1,40 @@
...
+| Github ID   | Name          | Organization |
+| ----------- | ------------- | ------------ |
+| Anilople    | Xiaoquan Wang | Some Bank    |
+| klboke      | Kailing Chen  | TapTap       |
+| lepdou      | Le Zhang      | Ant Group    |
+| nisiyong    | Stephen Ni    | Qihoo 360    |
+| vdisk-group | Lvqiu Ye      | Hundsun      |
+
+## Contributor
```
Comment (JaredTan95) [inline docs/zh/community/team.md:28]: `## 贡献者(Contributor)`

**Review by JaredTan95 — APPROVED** [review]

**Pull request merged**